h/static/scripts/test/time-test.js

```diff
@@ -50,18 +51,35 @@ describe('time', function () {
   });
 
   describe('.toFuzzyString', function () {
+
+    function mockIntl() {
```

Comment:
Not going to stop me merging but we generally refer to faked dependencies as fakes rather than mocks in the front-end code. Strictly speaking mocks have pre-programmed usage expectations and fakes do not, but the main reason is just consistency.